markup/house-style.md

```diff
@@ -75,6 +76,55 @@ We *don't* do this:
 ```html
 <meta charset="UTF-8" />
 ```
+### Form submit controls
+
+On modern browsers, `button type="submit"` provides a superset of functionality when compared to `input type=submit`.
+
+`button` elements are much easier to style than `input` elements. You can add
+inner HTML content (think `<em>`, `<strong>`, or even `<img>`), and use
+`::after` and `::before` pseudo-elements to achieve complex rendering while
+`input` only accepts a textual value attribute.
+
+Make sure to specify `name` and `value` attributes to maximize browsers
+compatibility.
+
+We do this:
+
+```html
+<form>
+	<button type="submit" name="myButton" value="foo">Continue</button>
+</form>
+```
+
+or
+
+We *don't* do this:
+
+```html
+<form>
+	<input type="submit" value="Continue">
+</form>
+```
+
+**Caution**: There is a shortcoming for Internet Explorer when using the `button`
+element. Indeed it submits the text within the `button` element as its value in
+the form data rather than the value of its `value` attribute.
+This becomes problematic when using multiple submit buttons in a single form with each a different value and purpose. 
+
+Here are some suggestions to work around these:
+
+- Interfaces could be redesigned so multiple submit buttons are not required.
```

Comment:
```suggestion
- Redesign the interface so multiple submit buttons are not required.
```
This suggestion changes the sentence from the passive voice to the active voice 😄